compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -161,6 +161,16 @@ class SymbolReferenceTable
       startPCLinkageInfoSymbol,
       instanceShapeFromROMClassSymbol,
 
+      // common atomic primitives
```

Comment:
Please use "32Bit" and "64Bit" to refer to the size of data being moved rather than the more ambiguous "Int" and "Long".